Composer/packages/client/src/components/Header/index.jsx

```diff
@@ -35,8 +35,11 @@ export const Header = props => {
         src={composerIcon}
       />
       <div css={title}>{formatMessage('Bot Framework Composer')}</div>
-      <div css={divider} />
-      <div css={botName}>{props.botName}</div>
+      {props.botName && (
+          <div css={divider} />
+          <span css={botName}>{`${props.botName}(${props.locale}}`}</span>
```

Comment:
Incorrect closing tag
```suggestion
          <span css={botName}>{`${props.botName} (${props.locale})`}</span>
```